src/components/FontAwesomeIcon.js

```diff
@@ -150,17 +159,23 @@ export default defineComponent({
     const transform = computed(() => objectWithKey('transform', typeof props.transform === 'string' ? faParse.transform(props.transform) : props.transform))
     const mask = computed(() => objectWithKey('mask', normalizeIconArgs(props.mask)))
 
-    const renderedIcon = computed(() =>
-      faIcon(icon.value, {
+    const renderedIcon = computed(() => {
+      const iconProps = {
         ...classes.value,
         ...transform.value,
         ...mask.value,
         symbol: props.symbol,
-        title: props.title,
-        titleId: props.titleId,
         maskId: props.maskId
-      })
-    )
+      }
+
+      if (semver.lt(SVG_CORE_VERSION, ICON_PACKS_STARTING_VERSION)) {
+        // the title attribute will only apply to versions prior to version 7.0.0
+        iconProps.title = props.title
+        iconProps.titleId = props.titleId
+      }
```

Comment:
@robmadole ---
Title attributes only if versions prior to v7, correct ?

Reply:
Yessir!